[2024-04-23T15:14:00.828+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-23T15:14:00.854+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: model_pipeline.data_quality_check manual__2024-04-23T15:13:50.352085+00:00 [queued]>
[2024-04-23T15:14:00.865+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: model_pipeline.data_quality_check manual__2024-04-23T15:13:50.352085+00:00 [queued]>
[2024-04-23T15:14:00.866+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-23T15:14:00.882+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): data_quality_check> on 2024-04-23 15:13:50.352085+00:00
[2024-04-23T15:14:00.893+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1182) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-23T15:14:00.895+0000] {standard_task_runner.py:63} INFO - Started process 1191 to run task
[2024-04-23T15:14:00.895+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'model_pipeline', 'data_quality_check', 'manual__2024-04-23T15:13:50.352085+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline_dag.py', '--cfg-path', '/tmp/tmpbsq9jnkf']
[2024-04-23T15:14:00.897+0000] {standard_task_runner.py:91} INFO - Job 115: Subtask data_quality_check
[2024-04-23T15:14:00.961+0000] {task_command.py:426} INFO - Running <TaskInstance: model_pipeline.data_quality_check manual__2024-04-23T15:13:50.352085+00:00 [running]> on host ec31c9b90297
[2024-04-23T15:14:01.077+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='model_pipeline' AIRFLOW_CTX_TASK_ID='data_quality_check' AIRFLOW_CTX_EXECUTION_DATE='2024-04-23T15:13:50.352085+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-23T15:13:50.352085+00:00'
[2024-04-23T15:14:01.079+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-23T15:14:01.744+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-23T15:14:01.745+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-23T15:14:01.760+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=model_pipeline, task_id=data_quality_check, execution_date=20240423T151350, start_date=20240423T151400, end_date=20240423T151401
[2024-04-23T15:14:01.796+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-23T15:14:01.830+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-23T15:14:01.832+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
